pos_membership/static/src/xml/PartnerLine.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+    Copyright (C) 2022-Today GRAP (http://www.grap.coop)
+    @author Sylvain LE GAL (https://twitter.com/legalsylvain)
+    License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+-->
+<templates id="template" xml:space="preserve">
+
+    <t
+        t-name="PartnerLine"
+        t-inherit="point_of_sale.PartnerLine"
+        t-inherit-mode="extension"
+        owl="1"
+    >
+        <xpath expr="//button[@class='edit-partner-button']" position="after">
```

Comment:
In terms of UX, isn't it better to display the membership status in a new column, rather than below the "Details" button?


![pos_ui_partner_tree](https://user-images.githubusercontent.com/3407482/201692623-1a804ee8-cd3c-48d9-9cb6-837e2205414c.png)



// not blocking

Reply:
I remember that I tried, but was not a better choice. (don't remember exactly why).
propose to merge it as it. If people has better implementation, totally open to accept it.